### DISCUSSION TITLE
ESXi8 bios启动显示 ACPI 主板布局需要EFI

### DISCUSSION THREAD

**Discussion by TwoBall** [kkkgo/PaoPaoGateWay#12] [问和答] [answered]
我在ESXi8上死活启动不了，改bios启动显示 ACPI 主板布局需要EFI

**Comment by kkkgo** [comment]
可以支持的，不过之前是因为会增加启动时间和体积，然后去掉efi启动。按理说虚拟机平台应该不会不支持传统启动，可否给出运行的日志参考一下呢？可以使用GifCam来录制gif

**Reply by kkkgo** [reply]
如果真的有不支持的情况，我可以编译一个带efi引导的iso

**Reply by TwoBall** [reply, marked as answer]
感谢，我试了下所有的虚拟机选项，发现指定有些类型是可以使用bios启动的。只不过最常见的“其他Linux 5.X 64位”默认只能UEFI启动
